documentation/basic/updates.md

```diff
@@ -20,4 +20,4 @@ HACS will check for updates to every element:
 
 ## Manually trigger updates
 
-On each RepositoryView (the page with details about the element) there is a reload icon at the top-right corner, clicking that will reload it.
+On each RepositoryView (the page with details about the element), click the "kebab" menu in the top-right corner and choose "Redownload" from the dropdown.

```

Comment:
```suggestion
On each RepositoryView (the page with details about the element), click the "kebab" menu in the top-right corner and choose "Update information" from the dropdown.

```

This page is about data updates, not about upgrading files.

Reply:
Noted, thank you - it wasn't clear for me.